kafka-client/src/main/java/dev/responsive/kafka/internal/stores/CommitBuffer.java

```diff
@@ -523,7 +523,7 @@ private void doFlush(final long consumedOffset, final int batchSize) {
     flushSensor.record(1, endMs);
     flushLatencySensor.record(flushLatencyMs, endMs);
 
-    log.debug("Flushed {} records to remote in {}ms (offset={}, writer={}, numSubPartitions={})",
+    log.info("Flushed {} records to remote in {}ms (offset={}, writer={}, numSubPartitions={})",
```

Comment:
I found this log to be very useful to see how long a specific flush for X records takes so I added it back as info cc @ableegoldman